components/snowflake/sources/new-row/README.md

```diff
@@ -0,0 +1,43 @@
+# Snowflake New Row Source
+
+## Introduction
+
+This source emits a new event each time a new row is added to a specific table
+in Snowflake. It achieves so by periodically scanning such table for rows with a
+higher index than previous records, hence it is necessary for the target table
+to be indexed by a monotonically increasing column, or at least have a column
+with such a property so that this component can detect new rows.
+
+## Prerequisites
+
+To set up your Snowflake credentials, go to the [Pipedream
+Apps](https://pipedream.com/apps) page, and click on the **CONNECT AN APP**
+button. Look for the **Snowflake** app, and click on it. Fill in every field
+required in the, and click on **SAVE**.
+
+## Usage
+
+1. Visit the [Pipedream Sources](https://pipedream.com/sources) page and click
```

Comment:
Can just link to `https://pipedream.com/sources/new?key=snowflake-new-row` ?

Reply:
Nice, fixed